source/develop/release-management/features/storage/importstoragedomain.html.md

```diff
@@ -393,6 +393,37 @@ If the user want to register a specific floating disks in the system he should u
 
     curl -v -k -u "admin@redhat.com" -H "Content-type: application/xml" -d '<disk id="8ddb988f-6ab8-4c19-9ea0-b03ab3035347"><alias>dsdsdsdmap1_Disk3</alias> </disk>' "http://localhost:8080/ovirt-engine/api/datacenters/d2045b3a-a313-452f-8333-b1e0178a024e/storagedomains/60cec75d-f01d-44a0-9c75-8b415547bc3d/disks';'unregistered "
 
+### Troubleshooting
+
+#### Problem - Image path does not exist or cannot be accessed
+
+*   When registering entities such as VMs and Templates from an imported storage domain, one might encounter a failure when some of the volumes are missing in the storage domain.
+    The volumes might be missing from the following reasons:
+    1. Not all storage domains were imported to the engine, and some of the VM's/Template's disks were dependant on this storage domains.
+    2. The OVF_STORE disk was not synced when the disaster occured, causing the volumes in the VM/Template to be unsynced.
+    This might happen in numerous scenarios such as remove a snapshot, delete a disk or any other operation which changed the volume chain.
+
+    This type of failure will be logged in the engine, for example:
+
+    <I>ERROR [org.ovirt.engine.core.vdsbroker.irsbroker.GetImageInfoVDSCommand] (default task-47) [14de7d5f] IrsBroker::getImageInfo::Failed getting image info imageId='425daada-2d07-4d7f-9365-4674e2ef50a0' does not exist on domainName='data4', domainId='d07e18b6-20da-46b8-a7e5-5688dc3cb04a', error code: 'ImagePathError', message: Image path does not exist or cannot be accessed/created: (u'/rhev/data-center/mnt/10.35.16.43:_export_data4/d07e18b6-20da-46b8-a7e5-5688dc3cb04a/images/75a157ee-c485-423d-9c0e-62d5d3b9d718',)</I>
```

Comment:
This should probably be in a quote block, like so:

> ERROR [org.ovirt.engine.core.vdsbroker.irsbroker.GetImageInfoVDSCommand] (default task-47) [14de7d5f] IrsBroker::getImageInfo::Failed getting image info imageId='425daada-2d07-4d7f-9365-4674e2ef50a0' does not exist on domainName='data4', domainId='d07e18b6-20da-46b8-a7e5-5688dc3cb04a', error code: 'ImagePathError', message: Image path does not exist or cannot be accessed/created: (u'/rhev/data-center/mnt/10.35.16.43:_export_data4/d07e18b6-20da-46b8-a7e5-5688dc3cb04a/images/75a157ee-c485-423d-9c0e-62d5d3b9d718',)

(Basically, start the line with a `>`. You're not quoting someone, but you are quoting some<em>thing</em>.)